### PR TITLE
Applying suggestions from V1.0.0 review

### PR DESCRIPTION
**Pull request by pinin4fjords**
Addressing additional feedback in https://github.com/nf-core/differentialabundance/pull/44

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/differentialabundance/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Initial release of nf-core/differentialabundance, created with the [nf-core](https://nf-co.re/) template.
 
+### `Added
+
+- [[#1](https://github.com/nf-core/differentialabundance/issues/3)] - Set up in initial modules, define and validate workflow inputs
```
Comment (mashehu) [inline CHANGELOG.md:12]:
```suggestion
- [[#1](https://github.com/nf-core/differentialabundance/issues/3)] - Set up initial modules, define and validate workflow inputs
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -169,11 +172,11 @@
                 },
                 "filtering_min_proportion": {
                     "type": "number",
-                    "description": "A minimum proportion of observations that must pass the threshold. Overrides  minimum_samples"
+                    "description": "A minimum (between 0 and 1) of observations that must pass the threshold. Overrides  minimum_samples"
```
Comment (mashehu) [inline nextflow_schema.json:175]:
```suggestion
                    "description": "A minimum proportion of observations, given as a number between 0 and 1, that must pass the threshold. Overrides  minimum_samples"
```

**Review by mashehu — APPROVED** [review]
Just some small things left. Almost there 🙂

**Comment by pinin4fjords** [comment]
Sorry @mashehu , you can't have your default falses- `nf-core schema build` keeps stripping them!

**Comment by pinin4fjords** [comment]
Thanks very much for the review @mashehu

**Pull request merged**